timelock-impl/src/main/metrics/metadata.yml

```diff
@@ -0,0 +1,11 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.timelock.metrics'
+
+namespaces:
+  requestMetadata:
+    docs: Metrics tracking metadata presence in lock requests sent to TimeLock
+    metrics:
+      numChangeMetadata:
+        docs: Number of change metadata objects contained in a lock request
+        type: histogram
```

Comment:
Not sure about the best way to name these. I wanted to highlight that we are only collecting metrics on how much/what kind of metadata we receive as part of requests. If we do other metrics in the future (e.g., how much metadata is currently persisted in TimeLock), I'd add them as a separate namespace.

Reply:
In fact, that's exactly what I did